src/app/app.module.ts

```diff
@@ -27,6 +28,18 @@ import { InfobarComponent } from './templates/talking-heads/infobar/infobar.comp
 import { MainPageComponent } from './main-page/main-page.component';
 import { TimelineMapComponent } from './templates/timeline/timeline-map/timeline-map.component';
 import { TalkingHeadsMapComponent } from './templates/talking-heads/talking-heads-map/talking-heads-map.component';
+import { SlickCarouselModule } from 'ngx-slick-carousel';
+import { M6Component } from './maps/m6/m6.component';
+import { SeclayoutComponent } from './maps/m6/seclayout/seclayout.component';
+import { PrevDirective } from './maps/m6/directives/prev.directive';
+import { NextDirective } from './maps/m6/directives/next.directive';
+import { AccordionDirective } from './maps/m6/directives/accordion.directive';
```

Comment:
This directive was added but is not used anywhere...

Reply:
they used att line 69

Reply:
It's declared in line 69 - where is it actually used?